### DISCUSSION TITLE
Is there ingress gateway component ?

### DISCUSSION THREAD

**Discussion by Kiddinglife** [akkadotnet/akka.net#5394] [General]
Hey, Is there ingress gaetway similar to akka-java-http/nginx-ingress/istio-gateway that is used as an entry point for incoming traffics into akka cluster? Those traffics are more than http/grpc and can be any customized protocol.

**Comment by Aaronontheweb** [comment]
@Kiddinglife at the moment no there isn't - that library, I think, is meant to be used with Akka.Http which we don't really support at the moment.
